src/test/resources/html/framework/repeater.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<div>
+    <div id="testDiv1" class="test" onclick="clickDiv1()">Div1</div>
+    <div id="testDiv2" class="test" onclick="clickDiv2()">Div2</div>
+    <div id="testDiv3" class="test" style="display: none">Div3</div>
+    <div id="testDiv4" class="test">Div4</div>
+    <div id="testDiv5" class="test">Div5</div>
+</div>
+
+
+<script>
+
+    var clickCount = 0;
+
+    function clickDiv1() {
+        clickCount++;
+        console.log("Click count = " + clickCount);
+        if (clickCount === 7) {
+            document.getElementById("testDiv3").style.display = "block";
+
+        }
+    }
+
+    function clickDiv2() {
```

Comment:
this function is a duplicate of the above. They can be implemented as 1 function

Reply:
Done